.travis.yml

```diff
@@ -32,6 +32,7 @@ install:
   - sdkmanager --list | head -15
   - mkdir "$ANDROID_HOME/licenses" || true
   - echo -e "\n8933bad161af4178b1185d1a37fbf41ea5269c55" > "$ANDROID_HOME/licenses/android-sdk-license"
+  - echo -e "\n24333f8a63b6825ea9c5514f83c2829b004d1fee" > "$ANDROID_HOME/licenses/android-sdk-license"
```

Comment:
This doesn't use `>>`, which means it overwrites the file that the previous line writes. So, either the previous line should be removed, or this line should use `>>`.

Reply:
@zx2c4 Your right, I totally missed that. Weird that the build doesn't fail.

@pekingme I'll do a follow up PR for this.

Reply:
Addressed with #1739.